package.json

```
@@ -84,24 +84,25 @@
     "xmldom": "^0.1.27"
   },
   "devDependencies": {
-    "babel-core": "^7.0.0-bridge.0",
+    "@babel/runtime": "^7.4.3",
+    "babel-core": "^7.4.3",
```

They changed the package naming scheme (I think as part of 7.0.0 changes) so this should be @babel/core.

```suggestion
    "@babel/core": "^7.4.3",
```